packages/ripple-binary-codec/test/definitions.test.js

```diff
@@ -0,0 +1,71 @@
+const { encode, decode } = require('../dist')
+const { XrplDefinitions } = require('../dist/enums/xrpl-definitions')
```

Comment:
This should be importable directly from `../src/enums/xrpl-definitions` if you use TS

Reply:
I’m not sure in JS how, but yes we should try to avoid using dist

Reply:
It looks like the existing tests don't use dist anymore even in JS: https://github.com/XRPLF/xrpl.js/blob/main/packages/ripple-binary-codec/test/binary-json.test.js#L2
I assume these changes were made in your Jest PR and thus weren't there when Jackson originally started this PR.